### PR TITLE
[FIX] Replace '.at()'; fix default cookie value;

### PR DESCRIPTION
**Pull request by kovtun-d**

### CODE REVIEW

File: paf-mvp-frontend/src/utils/cookie.ts [hunk 1]
```diff
@@ -1 +1 @@
-export const getCookieValue = (name: string): string => document.cookie.match(`(^|;)\\s*${name}\\s*=\\s*([^;]+)`)?.pop() || '';
+export const getCookieValue = (name: string): string => document.cookie.match(`(^|;)\\s*${name}\\s*=\\s*([^;]+)`)?.pop() || undefined;
```
Comment (OlivierChirouze) [inline paf-mvp-frontend/src/utils/cookie.ts:1]: `: string | undefined`

Comment (OlivierChirouze) [reply paf-mvp-frontend/src/utils/cookie.ts:1]: I think it would be more strictly correct to use `??` operator to allow empty strings, WDYT?

**Review by OlivierChirouze — CHANGES_REQUESTED** [review]

File: paf-mvp-frontend/src/utils/cookie.ts (hunk 1 above)
Comment (kovtun-d) [reply paf-mvp-frontend/src/utils/cookie.ts:1]: I don't think it's possible to set an empty string in cookies.
https://stackoverflow.com/questions/8775572/setcookie-to-an-empty-value-not-working#:~:text=You%20can't%20set%20a,deleted%20from%20the%20remote%20client.

Comment (OlivierChirouze) [reply paf-mvp-frontend/src/utils/cookie.ts]: of course, you're right ;-)
Thanks

**Review by OlivierChirouze — APPROVED** [review]

**Pull request merged**